files/build_templates/events_info.json

```diff
@@ -5,10 +5,6 @@
             "name": "monit",
             "parse_json": "monit_regex.json"
         },
-        {
-            "name": "sshd",
```

Comment:
>sshd [](http://example.com/codeflow?start=21&length=4)

why remove? I cannot find info in PR description.

Reply:
Updated PR description, but that event is not being supported.

Reply:
I will close this comment. but still could you help me understand "sshd regex and zebra regex are removed because those events are currently not being supported." sshd does not support or regex does not support or eventd does not support?

Reply:
eventd will not support sshd
zebra regex is no longer supported.